js/adapt-contrib-gmcq.js

```diff
@@ -1,66 +1,66 @@
 define([
-    'core/js/adapt',
-    'components/adapt-contrib-mcq/js/adapt-contrib-mcq'
+  'core/js/adapt',
+  'components/adapt-contrib-mcq/js/adapt-contrib-mcq'
 ], function(Adapt, Mcq) {
 
-    var Gmcq = Mcq.view.extend({
+  var Gmcq = Mcq.view.extend({
 
-        setupQuestion: function() {
-            Mcq.view.prototype.setupQuestion.call(this);
+    setupQuestion: function() {
+      Mcq.view.prototype.setupQuestion.call(this);
 
-            this.listenTo(Adapt, {
-                'device:changed': this.resizeImage,
-                'device:resize': this.onDeviceResize
-            });
+      this.listenTo(Adapt, {
+        'device:changed': this.resizeImage,
+        'device:resize': this.onDeviceResize
+      });
 
-        },
+    },
 
-        onQuestionRendered: function() {
+    onQuestionRendered: function() {
 
-            this.resizeImage(Adapt.device.screenSize);
-            this.setUpColumns();
+      this.resizeImage(Adapt.device.screenSize);
+      this.setUpColumns();
 
-            this.$('label').imageready(this.setReadyStatus.bind(this));
+      this.$('.js-item-label').imageready(this.setReadyStatus.bind(this));
 
-        },
+    },
 
-        onDeviceResize: function() {
-            this.setUpColumns();
-        },
+    onDeviceResize: function() {
+      this.setUpColumns();
+    },
 
-        resizeImage: function(width) {
-            var imageWidth = width === 'medium' ? 'small' : width;
+    resizeImage: function(width) {
+      var imageWidth = width === 'medium' ? 'small' : width;
 
-            this.$('label').each(function(index) {
-                var $img = $(this).find('img');
-                var newSrc = $img.attr('data-' + imageWidth);
-                if (!newSrc) return;
-                $img.attr('src', newSrc);
-            });
+      this.$('.js-item-label').each(function(index) {
+        var $img = $(this).find('img');
+        var newSrc = $img.attr('data-' + imageWidth);
+        if (!newSrc) return;
+        $img.attr('src', newSrc);
+      });
 
-        },
+    },
 
-        setUpColumns: function() {
-            var columns = this.model.get('_columns');
+    setUpColumns: function() {
+      var columns = this.model.get('_columns');
 
-            if (!columns) return;
+      if (!columns) return;
 
-            if (Adapt.device.screenSize === 'large') {
-                this.$el.addClass('gmcq-column-layout');
-                this.$('.gmcq-item').css('width', (100 / columns) + '%');
-            } else {
-                this.$el.removeClass('gmcq-column-layout');
-                this.$('.gmcq-item').css('width', '');
-            }
-        }
+      if (Adapt.device.screenSize === 'large') {
+        this.$el.addClass('has-column-layout');
+        this.$('.js-mcq-item').css('width', (100 / columns) + '%');
```

Comment:
opportunity to `return` early and lose the `else` block

Reply:
Opportunity to lose the _if...else_ completely:
```js
var isLarge = Adapt.device.screenSize === 'large';

this.$el.toggleClass('has-column-layout', isLarge);
this.$('.js-mcq-item').css('width', isLarge ? (100 / columns) + '%' : '');
```

Reply:
Updated function with suggestion